src/app/components/app-header/app-header.component.ts

```diff
@@ -16,13 +16,14 @@ import { SearchApi } from "../../api/search.api";
 import AppValues from "../../common/app.values";
 import { BaseComponent } from "src/app/common/base.component";
 import { AccountInfo } from "src/app/api/kamu.graphql.interface";
+import { NgbTypeaheadSelectItemEvent } from "@ng-bootstrap/ng-bootstrap";
 
 @Component({
     selector: "app-header",
     templateUrl: "./app-header.component.html",
 })
 export class AppHeaderComponent extends BaseComponent {
-    @Input() public searchValue: DatasetIDsInterface = {
+    @Input() public searchValue: DatasetIDsInterface | string = {
```

Comment:
I think `searchValue` only needs to be a string, this original `DatasetIDsInterface` is not required, we only used this field as a 2-way binding variable for a text input field.